_layouts/default.html

```diff
@@ -16,12 +16,14 @@
     {% include navbar.html %}
     {{ content }}
   </main>
-
   {% include footer.html %}
 
   {% include analytics.html %}
   {% include scripts.html %}
 
+  {%- if  site.author.buymeacoffee -%}
+    <script data-name="BMC-Widget" data-cfasync="false" src="https://cdnjs.buymeacoffee.com/1.0.0/widget.prod.min.js" data-id="{{ site.author.buymeacoffee }}" data-description="Support me on Buy me a coffee!" data-message="" data-color="#FFDD00" data-position="Right" data-x_margin="16" data-y_margin="16"></script>
+  {%- endif -%}
```

Comment:
This button script was provided by the Buy me a coffee website, it's a default configuration, but we might want to let `data-color` be configurable as well as `data-description`.

Reply:
Can we do that in this PR?
We can create a key in `config.yml` with these properties.

Reply:
Done.